core-services/release-controller/_releases/release-okd-scos-4.y-next.json

```diff
@@ -11,9 +11,13 @@
   "verify":{
     "upgrade":{
       "upgrade":true,
-      "prowJob": {
-        "name": "release-openshift-okd-scos-installer-e2e-aws-upgrade-from-scos-next"
-      },
```

Comment:
Is loosing this formatting intentional?  Ultimately, it doesn't matter all that much, but it's inconsistent with the formatting in the vast majority of the other files.

Reply:
i compared it to the stable config here: https://github.com/openshift/release/blob/master/core-services/release-controller/_releases/release-ocp-4.y-stable.json and thought I would do the same. if it is inconsistent, I can revert it to the original format.

Reply:
fixed it to the original format.